packages/twenty-server/project.json

```diff
@@ -150,8 +151,8 @@
           "nx typeorm -- migration:run -d src/database/typeorm/metadata/metadata.datasource",
           "nx typeorm -- migration:run -d src/database/typeorm/core/core.datasource"
         ],
-        "parallel": false
-      }
+        "parallel": true
```

Comment:
Yes I wasn't sure this was a good idea... But do we want to allow interdependencies? Maybe not

Reply:
It could generate silent errors which are difficult to reproduce, is it where we'll gain the most ? I think refactoring database creation (drop + recreate and with plugins) can be a better trade-off here